sdk/forms/src/main/java/com/klaviyo/forms/InAppForms.kt

```diff
@@ -13,10 +19,11 @@ import com.klaviyo.core.safeApply
 fun Klaviyo.registerForInAppForms(
     config: InAppFormsConfig = InAppFormsConfig()
 ): Klaviyo = safeApply {
-    // Ensure we only ever register one KlaviyoWebViewClient instance
-    if (!Registry.isRegistered<KlaviyoWebViewClient>()) {
-        Registry.register<KlaviyoWebViewClient>(KlaviyoWebViewClient(config))
-    }
+    // Register IAF services
+    Registry.registerOnce<PresentationManager> { KlaviyoPresentationManager() }
```

Comment:
nit - could be clean to use an apply here
```suggestion
    Registry.apply{
       registerOnce<PresentationManager> { KlaviyoPresentationManager() }
       registerOnce<BridgeMessageHandler> { KlaviyoBridgeMessageHandler() }
       ...
       }
```